.circleci/workflows.yml

```diff
@@ -409,22 +396,9 @@ jobs:
             export IMAGE_TAG=graviteeio.azurecr.io/am-gateway:${TAG}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV            
             docker build -f docker/gateway/Dockerfile-dev --build-arg GRAVITEEAM_VERSION=${AM_VERSION} -t ${IMAGE_TAG} /tmp
-      - docker-scan
-      - run:
-          when: always
-          name: Generate last log lines for slack message
-          command: |
-            API_URL=$(echo $CIRCLE_BUILD_URL | cut -d/ -f4-7)
-            FAILED_STEP=$(curl "https://circleci.com/api/v1.1/project/${API_URL}?circle-token=${CIRCLE_API_TOKEN}" | jq '.steps | .[] | flatten | map(select(.name? == "Run Snyk")) | .[] | {allocation_id, step}')
-            ALLOCATION_ID=$(echo "${FAILED_STEP}" | jq -r '.allocation_id')
-            ALLOCATION_ID=$(echo $ALLOCATION_ID | cut -d' ' -f2)
-            STEP=$(echo "${FAILED_STEP}" | jq -r '.step')
-            STEP=$(echo $STEP | cut -d' ' -f2)
-            curl -o slack_response.txt "https://circleci.com/api/v1.1/project/${API_URL}/output/${STEP}/0?file=true&allocation-id=${ALLOCATION_ID}&circle-token=$CIRCLE_API_TOKEN"
-            sed -i 's/\r$//g' slack_response.txt
-            SLACK_MESSAGE=$(cat slack_response.txt | tail -n 15 | sed '$!s/$/\\n/' | tr -d '\n')
-            echo "export SLACK_MESSAGE='${SLACK_MESSAGE}'" >> $BASH_ENV     
-      - notify-snyk-result
+            SLACK_MESSAGE2="${SLACK_MESSAGE} ${IMAGE_TAG}, "
+            echo "export SLACK_MESSAGE='${SLACK_MESSAGE2}'" >> $BASH_ENV
+            snyk container test ${IMAGE_TAG} --app-vulns
```

Comment:
with this change, does it mean that the pipeline will fail here if there are some vuln issues ?
int the previous implementation using the snyk orb, we go to the next step in case of failure https://github.com/gravitee-io/gravitee-access-management/pull/1824/files#diff-7bf091450b804ea25dc3d11ac28c50202d19178f7b53f54a02ad59c946afa6a7L83